### PR TITLE
Add an action in the conformance CI to increase available disk space 

### PR DESCRIPTION
**Pull request by bzsuni**
**What type of PR is this?**
/kind test

**What this PR does / why we need it**:
Add an action in the conformance CI to increase available disk space to prevent test execution failures due to insufficient disk capacity.

ref #5774
https://github.com/kubeedge/kubeedge/actions/runs/11026517530/job/30634159887?pr=5774

As follows, there may be issues with test failures due to disk pressure：
```yaml
status:
            conditions:
            - lastProbeTime: null
              lastTransitionTime: "2024-09-25T05:44:10Z"
              message: '0/2 nodes are available: 1 node(s) had untolerated taint {node.kubeedge.io/conformance:
                remove-when-completed}, 1 node(s) had untolerated taint {node.kubernetes.io/disk-pressure:
                }. preemption: 0/2 nodes are available: 2 Preemption is not helpful for scheduling.'
              reason: Unschedulable
```

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
Before cleanup:
```bash
$ dh -h /

Filesystem      Size  Used Avail Use% Mounted on
/dev/root        73G   54G   20G  74% /
```

After cleanup:
```bash
+ df -Th
Filesystem     Type   Size  Used Avail Use% Mounted on
/dev/root      ext4    73G   28G   45G  39% /
```
**Does this PR introduce a user-facing change?**:

```release-note
none
```

### CODE REVIEW

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -441,11 +437,22 @@ jobs:
...
 
+        - name: Increase disk space
+          uses: fcbg-platforms/increase-disk-space@latest
+          with:
+            tool-cache: true
+            android: true
+            dotnet: true
+            haskell: true
+            large-packages: true
+            old-dotnet: true
```
Comment (fisherxu) [inline .github/workflows/main.yaml:454]: Any docs for what's meaning of these fields?

Comment (bzsuni) [reply .github/workflows/main.yaml:454]: On the official page https://github.com/jlumbroso/free-disk-space, there isn't a detailed explanation, and it only states "Most of the options are self-explanatory."
image


However, it seems that:
- android, dotnet, haskell refer to whether you want to clean up libraries related to Android, .NET, and Haskell, respectively.
- docker-images indicates whether you want to clean up Docker images.
- swap-storage refers to whether to disable the swap partition.
- The large-packages option is not very clear regarding which contents it cleans up, but it doesn't seem to negatively impact the conformance workflow.

Comment (fisherxu) [reply .github/workflows/main.yaml:454]: Thanks, we can add `docker-images` in future if the disk space is still not enough.

**Review by fisherxu — COMMENTED** [review]
/lgtm
/assign @Shelley-BaoYue @WillardHu

**Review by fisherxu — COMMENTED** [review]
/lgtm

**Pull request merged**